articles/anomaly-detection/references/attack-protection-triggers-actions.md

```diff
@@ -1,15 +1,17 @@
 ---
-title: Brute-Force Protection Triggers and Actions
-description: Brute-force protection triggers and actions taken upon anomaly detection and how blocks are cleared.
+title: Attack Protection Triggers and Actions
+description: Attack protection triggers and actions taken upon anomaly detection and how blocks are cleared.
 topics:
     - security
     - anomaly-detection
     - brute-force-protection
+    - credential-stuffing
+    - breached-password-protection
 contentType: reference
-useCase: customize-anomaly-detection
+useCase: anomaly-detection
 v2: true
 ---
-# Brute-Force Protection Triggers and Actions
+# Attack Protection Triggers and Actions
 
 ## 10 failed login attempts
```

Comment:
Feels like a strange heading.  Wouldn't this be something like "Single-User Brute Force"?  Might be helpful in the heading to signal the single vs mutli-user idea - that part isn't immediately clear until you parse through the details.